tests/spec/core/biblio-spec.js

```diff
@@ -85,7 +84,7 @@ describe("W3C — Bibliographic References", () => {
     let ref = doc.querySelector("#bib-TestRef1 + dd");
     expect(ref).toBeTruthy();
     // This prevents Jasmine from taking down the whole test suite if SpecRef is down.
-    if (!(await specRefPing).ok) {
+    if (!specRefOk.ok) {
```

Comment:
Sorry, this should just be `if(!specRefOk){  ... }`  as `specRefOk` is just a boolean and won't have a `.ok` member, right?